docs/es/week12/12-3.md

```diff
@@ -0,0 +1,546 @@
+---
+lang-ref: ch.12-3
+title: Atención y el Transformador
+lecturer: Alfredo Canziani
+authors: Francesca Guiso, Annika Brundyn, Noah Kasmanoff, and Luke Martin
+date: 21 Apr 2020
+lang: es
+translator: Willebaldo Gómez Jiménez
+translation-date: 14 Sep 2020
+---
+
+<!-- ## [Attention](https://www.youtube.com/watch?v=f01J0Dri-6k&t=69s) -->
+## [Atención](https://www.youtube.com/watch?v=f01J0Dri-6k&t=69s)
+
+<!-- We introduce the concept of attention before talking about the Transformer architecture. There are two main types of attention: self attention *vs.* cross attention, within those categories, we can have hard *vs.* soft attention. -->
+Presentamos el concepto de "atención" antes de hablar sobre la arquitectura del Transformador. Existen dos tipos principales de atención: auto atención y atención cruzada. Dentro de esas categorías, distinguimos entre atención "ténue" y atención "intensa".
```

Comment:
I used the terms "atención ténue" and "atención intensa" for "soft attention" and "hard attention", respectively, but I still have doubts about whether we should just use the English terms. Thoughts?

Reply:
In this context, soft means "smooth", hard means "1-hot". I think *suave* and *duro* are better choices. What do you think?

Reply:
Makes sense. Will push in a bit a change for this.